spec/models/rook_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+RSpec.describe Rook, type: :model do
+  describe "#valid_move?" do
+    let(:rook) { Rook.create(row: 0, col: 1) }
+
+    it "should allow legal vertical moves" do
```

Comment:
Im not sure I follow.  The test shows it returns false for an illegal move.  What is the difference changing it?  And, can you explain why it would matter if the test is similar to the king test?  They are using two different models, so they are still testing different models...